src/main/java/Duke.java

```diff
@@ -1,10 +1,177 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final int DONE_START = 5;
+    public static final int TODO_START = 5;
+    public static final int EVENT_START = 6;
+    public static final int DEADLINE_START = 9;
+
+    public static Task[] tasks = new Task[100];
```

Comment:
Try avoid using magic numbers

Reply:
Not a magic number if it is specified in the constants